src/components/Map.vue

```diff
@@ -50,7 +56,36 @@ export default {
         return [[bbox[1], bbox[2]], [bbox[3], bbox[0]]];
       }
       return null;
-    }
+    },
+    getBaseMap() {
```

Comment:
```suggestion
    baseMap() {
```

This is a computed property and usually you don't use get/set for them as they are not functions in the end... Then you'd neet to replace all getBaseMap occurances in the template, too.

Reply:
Fixed this. I also added a check to see if ssys should be used. I think this is a cleaner separation that builds off of your suggestion.